src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md

```diff
@@ -17,7 +17,8 @@ date: 2020-10-19
 {% capture page_hero_banner_content %}
 
 # User Experience Guidelines for Mobile Extensions
-Make your extension seamlessly integrate with Firefox for Android
+
+Make your extension seamlessly integrate with Firefox for Android.
```

Comment:
Nit, "seamlessly integrate" -> "seamlessly integrated"?

Reply:
@rpl because the extension is not becoming part of Firefox, it's something that runs in the context of Firefox, the original is correct.